src/auto_scheduler/compute_dag.cc

```diff
@@ -1023,7 +1032,10 @@ ComputeDAG ComputeDAG::RewriteLayout(Array<Step>* transform_steps,
             }
             original_compute_op = op;
             CHECK(!new_compute_op.defined());
-            new_compute_op = te::ComputeOp(pop->name, pop->tag, pop->attrs, pop->axis, new_body);
+            auto new_attrs = pop->attrs;
+            new_attrs.Set("ori_placeholder_layout", tvm::String(origin_layout));
+            new_attrs.Set("new_placeholder_layout", tvm::String(new_layout));
+            new_compute_op = te::ComputeOp(pop->name, pop->tag, new_attrs, pop->axis, new_body);
```

Comment:
I'm finding that we can get the updated layout information here, can this help us not to add extra things to structures like `Conv2DAttrs`? @merrymercy

Reply:
No. `Conv2DAttrs` is used to pass layout information across multiple relay passes.
The object created here cannot live across multiple relay passes.